odtp/helpers/models.py

```diff
@@ -0,0 +1,72 @@
+from pydantic import BaseModel, Field, validator
+from typing import List, Optional, Union
+
+
+class DataInput(BaseModel):
+    name: str
+    type: str
+    path: str
+    description: str
+
+
+class DataOutput(BaseModel):
+    description: str
+    name: str
+    path: str
+    type: str
+
+
+class Parameter(BaseModel):
+    name: str
+    datatype: str
+    description: str
+    default_value: Optional[str] = Field(alias="default-value")
+    options: Optional[List[str]] = None
+    allow_custom_value: Optional[bool] = Field(alias="allow-custom-value", default=None)
+
+
+class Port(BaseModel):
+    name: str
+    description: str
+    port_value: int = Field(alias="port-value")
+
+
+class Tool(BaseModel):
+    tool_author: str = Field(alias="tool-author")
+    tool_license: str = Field(alias="tool-license")
+    tool_name: str = Field(alias="tool-name")
+    tool_repository: str = Field(alias="tool-repository")
+    tool_version: str = Field(alias="tool-version")
+
+
+class Devices(BaseModel):
+    gpu: bool
+
+
+class OdtpDotYamlSchema(BaseModel):
```

Comment:
Could you adapt this part to the new schema provide here: https://github.com/odtp-org/odtp-component-template/blob/main/odtp.yml

Reply:
Feel free to ignore the data-input and data-output as I will rollback those sections.

Reply:
Sure I will adapt. It would be great if you can update first and then let me know, so I can test my adaptation before I merge